Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent {
+    label 'docker && linux'
+  }
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '5'))
+    timestamps()
+  }
```

Comment:
This is correct. I choose 5 out of nowhere. We could increase that in the future. 
Note that at this moment, we are not keeping the `jar` file we build. Because there is no really use for it on its own. However, we could `archive` it and then the storage question would be a good one to ask our-self.

Reply:
`timestamp` is prefixing all the build logs with the date-time of the log message. This is handy if later, we have multiple tests at the same time or if we want to see why a build failed and if it has anything to do with something happening on the infra at the same time.